Vagrantfile

```diff
@@ -244,7 +245,12 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
       os.image        = voc['image']          || creds['OSImage']    || /Fedora/         # Regex or String
       os.ssh_username = user = voc['ssh_user']|| creds['OSSshUser']  || "root"           # login for the VM instance
       os.server_name  = ENV['OS_HOSTNAME']    || vagrant_openshift_config['instance_name'] # name for the instance created
-      full_provision(override.vm, user)
+      floating_ip_pool = creds['OSFloatingIPPool'] || ENV['OS_FLOATING_IP_POOL']
```

Comment:
Possible but would 2 two very long lines. We need to have `:auto` sym and `false` as value, not string to get desired behavior. If you confirm I can make it 2 lines but would be hardly readable. Like:
```
os.floating_ip = (creds['OSFloatingIPPool'] || ENV['OS_FLOATING_IP_POOL']) == ":auto" ? :auto : (creds['OSFloatingIPPool'] || ENV['OS_FLOATING_IP_POOL'])
```
What do you suggest?